src/main/java/frc/robot/subsystems/LEDs.java

```diff
@@ -73,6 +71,14 @@ public void clearAnimation() {
     CANdle.setLEDs(0, 0, 0, 0, 0, 0);
   }
 
+  /**
+   * Clears a specified chunk of LEDs of the current animation or color
+   */
+  public void clearAnimationChunk(int startIdx, int count) {
+    setLEDBrightness(0);
```

Comment:
You don't need to setLEDBrightness to 0. You already cleared the LED color when you set the RGB values to 0. If you set the LED Brightness to zero, this'll turn off the LEDs completely 🥺